eda/setup.py

```diff
@@ -15,7 +15,7 @@
 ###########################
 
 version = ag.load_version_file()
-version = ag.update_version(version, use_file_if_exists=False, create_file=True)
+version = ag.update_version(version)
```

Comment:
Why change?

Reply:
That was a mistake. Version file should be created only in `core`